### DISCUSSION TITLE
Live Chat Delays on High-Traffic Website: Seeking Solutions

### DISCUSSION THREAD

**Discussion by LydiaCotter** [community/community#146330] [Programming Help]
### Select Topic Area

Question

### Body

Hello everyone,

I’m facing an issue with the live chat feature on my website, which provides custom packaging services. At times, during periods of high traffic, live chat messages are received with significant delays or sometimes customers leave before we can respond.

We’ve installed a live chat system but we’ve noticed that when traffic spikes, the chat notifications are delayed and this leads to missed opportunities as customers leave the site.

Here’s a bit more about the situation:

The website experiences high traffic at times and this seems to be impacting the chat service’s response time.
I’ve already optimized the website for speed but the issue persists during peak times.
I’m unsure if it’s a limitation of the chat software, server performance or something else entirely.
I’m looking for advice or recommendations on:

How to optimize live chat for high traffic websites.
Whether I should consider changing my current live chat software or making adjustments to improve performance.
Any insights on managing live chat during high traffic loads.
If anyone has experience with similar challenges or can suggest tools and strategies to improve live chat performance, I’d really appreciate your input!

For reference, here’s my website: https://customdonutboxes.com/

Thank you in advance!

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @LydiaCotter!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
